tests/test_datetime.rs

```diff
@@ -0,0 +1,108 @@
+#![feature(concat_idents)]
+
+extern crate pyo3;
+
+use pyo3::prelude::*;
+
+use pyo3::ffi::*;
+
+fn _get_subclasses<'p>(
+    py: &'p Python,
+    py_type: &str,
+    args: &str,
+) -> (&'p PyObjectRef, &'p PyObjectRef, &'p PyObjectRef) {
+    macro_rules! unwrap_py {
```

Comment:
You can get rid of that macro by unwrapping the import directly and replacing the macro with 

```rust
    let unwrap_py = |e: PyResult<_>| e.map_err(|e| e.print(*py)).unwrap();
```